firebase/package.json

```diff
@@ -20,7 +20,7 @@
   "engines": {
     "node": "16"
   },
-  "main": "lib/index.js",
+  "main": "lib/firebase/functions/index.js",
```

Comment:
One way to avoid these unnecessary nested folders is to only import within the `firebase` project since it is technically its own project.